core/jobs/batch_jobs/contributor_admin_stats_jobs.py

```diff
@@ -93,6 +93,8 @@ def run(self) -> beam.PCollection[job_run_result.JobRunResult]:
                 ndb_io.GetModels(
                 suggestion_models.TranslationContributionStatsModel.get_all(
                     include_deleted=False))
+            | 'Filter translation contribution with no topic' >> beam.Filter(
+                lambda m: m.topic_id != '')
```

Comment:
Rather checking for empty topic ID we should validate whether the topic ID is correct. Also we should do this for all 4 types of stats.

Reply:
Just need some clarifications here. If the topic is deleted should we consider counts of those topics or should we modify them as well. (Here by counts I mean counts of submitted/rejected and so on contributions).

I think even if the topic is deleted we shouldn't modify it's count. Cause the stats represents potentiality of a user, if a user has contributed too much to a single topic and then we decide to remove that topic, his stats will eventually look bad even though he did good amount of contribution. Not sure tho :)

cc: @seanlip @sagangwee @qinghaoyang .

Reply:
Hm, it's a good question. I think it's probably fine to only limit to valid topics. It will be very rare (if ever) that we delete a topic, and for stats stuff we do need a clear way to regenerate stats from scratch at any time. So I suggest keeping it simple and only including existing topics. (I assume we can change this in the future and regenerate if needed.)

Just my 2c.

Reply:
I agree with Sean's reasoning.